extensions/hibernate-orm/deployment/src/test/java/io/quarkus/hibernate/orm/deployment/ClassNamesTest.java

```diff
@@ -55,12 +52,12 @@ public static void index() throws IOException {
         hibernateIndex = IndexingUtil.indexJar(determineHibernateJarLocation());
     }
 
-    @ParameterizedTest
-    @MethodSource("provideConstantsToTest")
-    void testClassNameRefersToExistingClass(DotName constant) {
-        assertThatCode(() -> getClass().getClassLoader().loadClass(constant.toString()))
-                .doesNotThrowAnyException();
-    }
+    //    @ParameterizedTest
+    //    @MethodSource("provideConstantsToTest")
+    //    void testClassNameRefersToExistingClass(DotName constant) {
+    //        assertThatCode(() -> getClass().getClassLoader().loadClass(constant.toString()))
+    //                .doesNotThrowAnyException();
+    //    }
```

Comment:
Alternatively, just have another `ClassNames` class in the Hibernate Reactive extension, with class names specific to Hibernate Reactive. And introduce a `ClassNamesTest` in the Hibernate Reactive extension as well :)